mlir/lib/Dialect/Math/Transforms/ExpandPatterns.cpp

```diff
@@ -202,6 +202,48 @@ static LogicalResult convertCeilOp(math::CeilOp op, PatternRewriter &rewriter) {
   rewriter.replaceOp(op, ret);
   return success();
 }
+
+// Convert `math.fpowi` to a series of `arith.mulf` operations.
+// If the power is negative, we divide the result by 1.
```

Comment:
is this supposed to say divide one by the result?

Reply:
Yes. Thanks for pointing this out.